lambdas/main.go

```diff
@@ -314,10 +319,30 @@ func getClusterStatus(ctx context.Context, stateTable, stateTableHashKey, stateK
 	if managementLambdaName == "" {
 		return protocol.ClusterStatus{}, fmt.Errorf("MANAGEMENT_LAMBDA is not set")
 	}
+
+	useSecretManagerEndpoint, err := strconv.ParseBool(os.Getenv("USE_SECRETMANAGER_ENDPOINT"))
+	if err != nil {
+		log.Warn().Msg("Failed to parse USE_SECRETMANAGER_ENDPOINT, assuming false")
+	}
+	var username, password string
+	if !useSecretManagerEndpoint {
+		log.Info().Msg("Secret manager endpoint not in use, sending credentials in body")
+		usernameId := os.Getenv("USERNAME_ID")
+		deploymentPasswordId := os.Getenv("DEPLOYMENT_PASSWORD_ID")
+		adminPasswordId := os.Getenv("ADMIN_PASSWORD_ID")
+		creds, err := common.GetDeploymentOrAdminUsernameAndPassword(usernameId, deploymentPasswordId, adminPasswordId)
+		if err == nil {
```

Comment:
if err!=nil we can already return at this point

Reply:
Changed